### ISSUE TITLE
The format of Renewal emails.

### ISSUE THREAD

**Issue by elainetao829** [d33pspace/RenewalWebsite#96]
Hi @d33pspace,
I compared the subscription email, the reset password email, and the check invoice confirm email. Here are two differences (valid for both English and Chinese),
1. The position of Renewal logo and button. 
Subscription email: logo and button are left aligned (same as the content)
Reset and invoice confirm emails: logo and button are in the middle 

2. The button style/format
Subscription email: the button is larger with margin 
Reset and invoice confirm emails: no margin 

![image](https://user-images.githubusercontent.com/42002784/45255701-2540e180-b3bd-11e8-9138-f2d802768c00.png)

![image](https://user-images.githubusercontent.com/42002784/45255704-3be73880-b3bd-11e8-8c15-0ef9e98e7e82.png)

![image](https://user-images.githubusercontent.com/42002784/45255710-46a1cd80-b3bd-11e8-8dff-11ffc9909c6f.png)

**Comment by kolte** [comment]
Changes done in reset password and history confirmation email but subscription email might be sent from zapier side, can you confirm? @d33pspace 

@elainetao829  other two are ready for testing

**Comment by d33pspace** [comment]
Yes, the subscription email is sent by Drip. This issue covers only the reset password and email confirmation for giving history.

**Comment by d33pspace** [comment]
@elainetao829 This is ready for you to test the password reset and view giving history confirmation emails.

**Comment by elainetao829** [comment]
Verified and confirm closed.

**Comment by elainetao829** [comment]
Hi @d33pspace 
I just realized the left aligned problem is not fixed. Could you please confirm whether you want to change that?
The problem is, the logo and button in the 1st email (subscription) is left aligned, while in the latter emails they are in the middle.
